owncloudData/src/test/java/com/owncloud/android/data/user/datasources/implementation/OCRemoteUserDataSourceTest.kt

```diff
@@ -55,7 +55,7 @@ class OCRemoteUserDataSourceTest {
         used = OC_USER_QUOTA.used,
         free = OC_USER_QUOTA.available,
         relative = OC_USER_QUOTA.getRelative(),
-        total = OC_USER_QUOTA.getTotal()
+        total = OC_USER_QUOTA.total
```

Comment:
Yes, as long as it is not null... Be careful here as well

Reply:
`getTotal()` is used in this case 😄